bin/compile

```diff
@@ -35,23 +35,19 @@ ENV_DIR=$3
 # Export Path variables, for use in sub-scripts.
 export BUILD_DIR CACHE_DIR ENV_DIR
 
-# Set the Buildpack's internet target for downloading Python distributions.
-# The user can provide BUILDPACK_VENDOR_URL to specify a custom target.
```

Comment:
Don't we use `BUILDPACK_VENDOR_USE` to set the `dds-vendor` container as the runtime vendor? If this logic changes we will probably have to update that logic in places like https://github.com/plotly/dash-deployment-server/blob/master/install/var/lib/dokku/plugins/available/plotly/docker-args-deploy#L49

Reply:
Yeah, seems like we need to add the `BUILDPACK_S3_BASE_URL` as dds-vendor IP here: https://github.com/plotly/dash-deployment-server/blob/master/install/var/lib/dokku/plugins/available/plotly/docker-args-deploy#L49